lib/diagnostic.md

```diff
@@ -62,5 +72,7 @@ item elements) to the body DOM node, then hides it.
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
-
+```
+$(div).on("click", function)
+```
```

Comment:
you didn't open your function block, but I saw in your message that you didn't get around to adding some formatting